PP-8800: get solution licenses by status (#1410)
livechat/livechat-public-docs: src/pages/monetization/partner-program-api/index.mdx

- [Create License](#create-license)
- [Get Solution Licenses](#get-solution-licenses)

> - [Get Solution Licenses by status](#get-solution-licenses-by-status)

Reviewer: ```suggestion
- [Get Solution Licenses by Status](#get-solution-licenses-by-status)
```

Reviewer: Just to keep it consistent with the naming of other methods 😌

</Section>

<Section>

<Text>

> ## Get Solution Licenses by status

Reviewer: ```suggestion
## Get Solution Licenses by Status
```

Reviewer: same as above

`GET /v2/partners/solutions/licenses/<status>` - get the list of licenses by status

#### Headers

- `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`

#### Path Parameters

- `<status>` - **required** - license status, one of: `paid`, `trial`, `expired`

#### Query Parameters

- `date_from` - **required** - get a report for licenses created from a given date
- `date_to` - **required** - get a report for licenses created up to a given date

> - `limit` - an amount of licenses in response (min. 1, max. 15 - default)

Reviewer: Is 15 the default value? I'm asking as it's not entirely clear to me here :D

Author: Yes, it's the default :) I changed it, hope it's more clear - https://github.com/livechat/livechat-public-docs/pull/1410/commits/ea0961a8ea444c96c9ebf419367b3626139c67dd#diff-1fdbc72100416ca7bcf4938c953d51e3a2520b5431ba09b1205cf0b177de152fR2715

> - `limit` - an amount of licenses in response (min. 1, max. 15 - default)

Reviewer: ```suggestion
- `limit` - the amount of licenses in response (min. 1, max. 15 - default)
```

> - `next_cursor` - a next page token received in a previous request

Reviewer: ```suggestion
- `next_cursor` - a next page token received from the previous request's response
```

**Note:** Date format: `YYYY-MM-DDTHH:mm:ssZ`

#### Response

> - `200 - OK` - success, see the example on the right

Reviewer: ```suggestion
- `200 - OK` - success, see the example below
```
